lectures/week8/spark_streaming.py

```diff
@@ -0,0 +1,6 @@
+from pyspark import SparkContext
+from pyspark.streaming import StreamingContext
+
+# Create a local StreamingContext with two working thread and batch interval of 1 second
+sc = SparkContext("local[2]", "NetworkWordCount")
+ssc = StreamingContext(sc, 1)
```

Comment:
please change the name to be more meaningful.

Reply:
ok